src/components/MovieInfo.tsx

```diff
@@ -30,6 +34,7 @@ const MovieInfo: React.FC<MovieInfoProps> = ({ data }) => {
                 <div className="card-body">
                   <div className="card-title lead text-uppercase">{Title}</div>
                   <p className="card-text">Released on: {Year}</p>
+                  <p className="card-text">{icon} {Type}</p>
```

Comment:
We could always take advantages of the conditional rendering and add something like this instead:
```
    <p className="card-text text-warning align-items-center d-flex justify-content-center">
                    {Type === "movie" ? (
                      <BiCameraMovie />
                    ) : (
                      <PiTelevisionSimple />
                    )}
                    <span className="mx-1 text-info text-capitalize fs-6">
                      {Type}
                    </span>
                  </p>
```